### DISCUSSION TITLE
.bin file

### DISCUSSION THREAD

**Discussion by L-Delorme** [erichelgeson/BlueSCSI-v1#13] [General]
Hi @erichelgeson 
would you mind uploading a .bin for those of us wanting to flash the STM32 with STM32CubeProgrammer? 
Thanks!

**Comment by erichelgeson** [comment]
Hi, Yes I'm working on updating some builds and will post them soon.

**Reply by erichelgeson** [reply]
You can find the latest `bin` on the releases page now https://github.com/erichelgeson/BlueSCSI/releases

**Reply by L-Delorme** [reply]
Thanks!
